d2l-tspan.html

```diff
@@ -16,14 +16,34 @@
 				border: none;
 			}
 
-			#float {
+			.float,
```

Comment:
Note - our practice is to avoid generic class names because when shady-DOM is used, the elements (in this component) with these class names are potential target of selectors using the same name classes from other libraries.  In other words, styles could bleed to these elements from elsewhere.  We've seen this happen previously.  Our practice when using CSS class names is to name them `.d2l-component-name-*`.